test/unit/mdc-select/mdc-select-enhanced.test.js

```diff
@@ -967,3 +991,37 @@ test('#constructor instantiates a leading icon if an icon element is present', (
   assert.isTrue(root.classList.contains(cssClasses.WITH_LEADING_ICON));
   document.body.removeChild(menu);
 });
+
+test('#constructor instantiates the helper text if present', () => {
```

Comment:
Is there a reason these don't use `setupTest` while `setupTest` was seemingly updated to support helper text?

In particular would that make it possible to just reference `helperText` directly from `setupTest`'s return value in the destroy test?

(Ditto for mdc-select.test.js)